README.md

```diff
@@ -593,22 +596,23 @@ The session can then be authenticated or invalidated from the host app or any
 of the engines and the state will be synchronized via the service.
 
 One thing to be aware of is that if the authentication route is outside of the
-engine (e.g. in the host app), it is necessary to override the
-`triggerAuthentication` method of the `AuthenticatedRouteMixin` inside of the
-engine as that needs to transition to an **external** route in that case:
+engine (e.g. in the host app), it is necessary to use the special
+`transitionToExternal` method in the engine to transition to it. That can be
+done by passing a callback instead of a route name to the session service's
+`requireAuthentication` method in that case:
 
 ```
 // my-engine/addon/routes/index.js
 import Route from '@ember/routing/route';
-import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import { inject as service } from '@ember/service';
 
-export default class IndexRoute extends Route.extend(AuthenticatedRouteMixin) {
-  triggerAuthentication() {
-    this.transitionToExternal('login');
-  }
-}
+export default class IndexRoute extends Route {
+  @service session;
 
-```
+  beforeModel(transition) {
+    this.get('session').requireAuthentication(transition, () => this.transitionToExternal('login'));
+  },
+}
```

Comment:
Can't add the suggestion but missing a closing triple ` here

Reply:
fixed 👍